app/src/main/kotlin/com/hippo/ehviewer/client/data/GalleryInfo.kt

```diff
@@ -142,5 +144,4 @@ fun GalleryInfo.findBaseInfo(): BaseGalleryInfo = when (this) {
 
 fun GalleryInfo.asGalleryDetail(): GalleryDetail? = this as? GalleryDetail
 
-val GalleryInfo.hasAds
-    get() = simpleTags?.any { "extraneous ads" in it } ?: false
+fun detectAds(index: Int, size: Int) = index > size - 10 && Settings.stripExtraneousAds.value
```

Comment:
We can merge without enabling it If you insist, let this always return false.
Also nit: this file is not a well place for this function.

Reply:
Why we not enabling it?
It can block some ADs at least and do no harm.

Reply:
Done